common/src/test/java/org/opensearch/ml/common/utils/StringUtilsTest.java

```diff
@@ -117,6 +120,21 @@ public void getParameterMap() {
         assertEquals("[1.01,\"abc\"]", parameterMap.get("key5"));
     }
 
+    @Test
+    public void getInterfaceMap() {
+        final Set<String> allowedInterfaceFieldNameList = new HashSet<>(Arrays.asList("input","output"));
+        Map<String, Object> parameters = new HashMap<>();
+        parameters.put("input", "value1");
+        parameters.put("output", 2);
+        parameters.put("key3", 2.1);
+        parameters.put("key4", new int[]{10, 20});
+        parameters.put("key5", new Object[]{1.01, "abc"});
+        Map<String, String> interfaceMap = StringUtils.filteredParameterMap(parameters, allowedInterfaceFieldNameList);
+        Assert.assertEquals(2, interfaceMap.size());
+        Assert.assertEquals("value1", interfaceMap.get("input"));
```

Comment:
This use case is too simple. Add some real json schema use case for the input/output ?

Reply:
This method only filtered keys.